v2/sourcedb-to-spanner/README.md

```diff
@@ -60,18 +60,18 @@ mvn test
 ### Executing Template
 
 #### Required Parameters
-* **sourceDbURL** (JDBC URL of the source database): The URL which can be used to connect to the source database. (Example: jdbc:mysql://10.10.10.10:3306/testdb)
+* **sourceConfigURL** (Configuration to connect to the source database): Can be the JDBC URL or the location of the sharding config. (Example: jdbc:mysql://10.10.10.10:3306/testdb or gs://test1/shard.conf)
 * **username** (username of the source database): The username which can be used to connect to the source database.
 * **password** (username of the source database): The username which can be used to connect to the source database.
 * **instanceId** (Cloud Spanner Instance Id.): The destination Cloud Spanner instance.
 * **databaseId** (Cloud Spanner Database Id.): The destination Cloud Spanner database.
 * **projectId** (Cloud Spanner Project Id.): This is the name of the Cloud Spanner project.
-* **DLQDirectory** (GCS path of the dead letter queue directory): The GCS path of the DLQ Directory to be used during migrations
+* **OutputDirectory** (GCS path of the output directory): The GCS path of the directory where all errors and skipped events are dumped to be used during migrations
 
 #### Optional Parameters
 * **jdbcDriverJars** (Comma-separated Cloud Storage path(s) of the JDBC driver(s)): The comma-separated list of driver JAR files. (Example: gs://your-bucket/driver_jar1.jar,gs://your-bucket/driver_jar2.jar).
 * **jdbcDriverClassName** (JDBC driver class name): The JDBC driver class name. (Example: com.mysql.jdbc.Driver).
-* **tables** (Comma seperated list of tables to migrate): Tables that will be migrated to Spanner. Leave this empty if all tables are to be migrated. (Example: table1,table2).
+* **tables** (Colon seperated list of tables to migrate): Tables that will be migrated to Spanner. Leave this empty if all tables are to be migrated. (Example: table1:table2).
```

Comment:
Another alternative is '|' as a delimiter. Okay with either.

Reply:
Should we just keep it a comma and think about Terraform based runs here, where this will not be a concern?

A user will likely not create a big gCloud command on their own, and whatever documentation they will look at to build the command can highlight the escape character that's needed for gCloud.

We should emphasize readability here, and colon, pipe are less readable.

Reply:
Both work in this case. You could use comma or colon, but we advertise colon as  something that works in both scenarios. The issue with the command on gcloud is already something which has been raised as a concern, so allowing for an alternate here.